test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -83,5 +85,34 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
         }
+
+        [Theory, TraceTest]
+        [MemberData(nameof(DnxSdks))]
+        public void AssemblyInfo(DnxSdk sdk)
+        {
+            // Arrange
+            var solution = TestUtils.GetSolution<DnuPackTests>(sdk, "AssemblyInfo");
+            var project = solution.GetProject("Test");
+
+            sdk.Dnu.Restore(solution.RootPath).EnsureSuccess();
+
+            // Act
+            var result = sdk.Dnu.Pack(project.ProjectDirectory);
+
+            // Assert
+            Assert.Equal(0, result.ExitCode);
+
+            var assemblyPath = Path.Combine(result.RootPath, "Debug", "dnx451", "Test.dll");
+            var assembly = Assembly.LoadFrom(assemblyPath);
```

Comment:
We should use `System.Reflection.Metadata` instead of using `Assembly.Load`. You can't load this assembly with the same name twice.